docker/drake/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# -*- mode: dockerfile -*-
+# vi: set ft=dockerfile :
+
+# git clone -b gcs-science-robotics git@github.com:wrangelvid/drake.git
+# mkdir drake-build
+# cd drake-build
+# cmake -DWITH_MOSEK=ON [-DWITH_GUROBI=ON -DWITH_ROBOTLOCOMOTION_SNOPT=ON] ../drake
+# make -j
+# Then copy this Dockerfile into the drake-build folder
+# docker build -f Dockerfile -t wrangelvid/drake:gcs-science-robotics ./install
+#
+# docker login
+# docker push wrangelvid/drake:gcs-science-robotics
+
+FROM ubuntu:jammy
+ARG DEBIAN_FRONTEND=noninteractive
+ENV SHELL /bin/bash
+# Open ports for meshcat:
+EXPOSE 7000-7099/tcp
+EXPOSE 8888/tcp
+
+LABEL org.opencontainers.image.authors="David von Wrangel"
+LABEL org.opencontainers.image.description="Drake with sampling based planning for the gcs paper"
+LABEL org.opencontainers.image.licenses="BSD-3-Clause"
+LABEL org.opencontainers.image.source="https://github.com/wrangelvid/drake/tree/gcs-science-robotics"
+LABEL org.opencontainers.image.vendor="Massachusetts Institute of Technology"
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+ENV TZ=America/New_York
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# Install curl and useful transport
+RUN apt-get update && yes "Y" \
+      | apt-get install --no-install-recommends curl apt-transport-https
+RUN apt install -y tzdata
+COPY . /opt/drake/
+
+# Install drake prereqs.
+RUN apt-get update \
+  && yes "Y" | bash /opt/drake/share/drake/setup/install_prereqs
+
+# Install deepnote dependencies.
```

Comment:
strange that these are in drake, and note the deepnote dockerfile?

do we even need two different dockerfiles?  i would think just managing one, and hosting it on wrangelvid, would be enough?

Reply:
Working on that right now. After your first comment I noticed that we can just put both of the docker files together :)